### ISSUE TITLE
SettingsCard missing invoke pattern

### ISSUE THREAD

**Issue by AmelBawa-msft** [CommunityToolkit/Windows#391]
### Issue
Accessibility Insights reporting: "A button must support one of these patterns: Invoke, Toggle, or ExpandCollapse." when using a SettingsCard that is clickable (`IsClickEnabled=True`).

### Investigation
- `SettingsCardAutomationPeer` may possibly need to implement `GetPatternCore(PatternInterface patternInterface)` method.
```cs
protected override object GetPatternCore(PatternInterface patternInterface)
{
    return patternInterface == PatternInterface.Invoke ? this : base.GetPatternCore(patternInterface);
}
```
- Implement `IInvokeProvider` in the same class (since we're returning `this`).
- Or instead, maybe the `SettingsCardAutomationPeer` can inherit from `ButtonBaseAutomationPeer`?

### Steps to reproduce

```text
- Add a settings card

<controls:SettingsCard Header="Example" IsClickEnabled="True" />


- Run accessibility scan on the displayed card
- The following error is reported: "A button must support one of these patterns: Invoke, Toggle, or ExpandCollapse."
```

### Expected behavior

No errors

### Code Platform

- [ ] UWP
- [X] WinAppSDK / WinUI 3
- [ ] Web Assembly (WASM)
- [ ] Android
- [ ] iOS
- [ ] MacOS
- [ ] Linux / GTK

### Windows Build Number

- [ ] Windows 10 1809 (Build 17763)
- [ ] Windows 10 1903 (Build 18362)
- [ ] Windows 10 1909 (Build 18363)
- [ ] Windows 10 2004 (Build 19041)
- [ ] Windows 10 20H2 (Build 19042)
- [ ] Windows 10 21H1 (Build 19043)
- [ ] Windows 10 21H2 (Build 19044)
- [ ] Windows 10 22H2 (Build 19045)
- [X] Windows 11 21H2 (Build 22000)
- [ ] Other (specify)

### App minimum and target SDK version

- [ ] Windows 10, version 1809 (Build 17763)
- [ ] Windows 10, version 1903 (Build 18362)
- [ ] Windows 10, version 1909 (Build 18363)
- [ ] Windows 10, version 2004 (Build 19041)
- [ ] Windows 10, version 2104 (Build 20348)
- [ ] Windows 11, version 22H2 (Build 22000)
- [ ] Other (specify)

### Help us help you

Yes, I'd like to be assigned to work on this item.

**Comment by michael-hawker** [comment]
Thanks @AmelBawa-msft for the report and offering to help. We're working towards an 8.1 release in the near future. This seems like a simple fix, if you can open a PR this week, we can get look to get it into the upcoming release.

**Comment by niels9001** [comment]
@AmelBawa-msft Just wanted to check; were you planning to do a PR with these fixes?

**Comment by AmelBawa-msft** [comment]
@niels9001, unfortunately I will not be working on this PR. At least not in the near future. If anyone would like to load balance, please feel free to do so ❤️

**Comment by g0g8rz** [comment]
Just to add some more support for this issue, it seems this might be a blocker for software trying to comply with the [European accessibility act](https://commission.europa.eu/strategy-and-policy/policies/justice-and-fundamental-rights/disability/union-equality-strategy-rights-persons-disabilities-2021-2030/european-accessibility-act_en) ([wikipedia summary](https://en.wikipedia.org/wiki/European_Accessibility_Act)) which takes full effect June 2025.  From my understanding (although I am not a lawyer, nor a European) software must comply with [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG20/) and UIs containing the SettingsCard fails [WCAG 4.1.2](https://www.w3.org/TR/WCAG20/#ensure-compat-rsv)

![Image](https://github.com/user-attachments/assets/e72bec24-7513-4837-b9ec-9045e78ef525)
